src/lib/pages/backoffice/Actions/CheckIn/CheckedInItems/CheckedInItems.js

```diff
@@ -12,7 +12,8 @@ export default class CheckedInItems extends Component {
 
   renderItems = () => {
     const { items } = this.props;
-    const entries = items.map((item) => {
+
+    const entries = items.reverse().map((item) => {
```

Comment:
This is probably ok in most of the cases, but it might not be enough.
Here you are reversing the list client side, in place. The issue is that Elasticsearch will only give you the first `x` results. If the item to have as first is at the `x+1` position, the code above will not work because Elasticsearch did not return it.

It would be better to change the Elasticsearch query sort order instead, if possible.